packages/ui-signer/src/Unlock.tsx

```diff
@@ -14,14 +14,17 @@ import keyring from '@polkadot/ui-keyring/index';
 import translate from './translate';
 
 type Props = I18nProps & {
-  error?: string,
+  autoFocus?: boolean,
+  error?: UnlockI18n | null,
```

Comment:
Restore as it was.

Reply:
I'll be passing through InputErrorMessage type now